windows-msi/doc/config/Where are my config files.txt.in

```diff
@@ -0,0 +1,6 @@
+﻿The @PRODUCT_NAME@Service has switched to a separate configuration
+directory.
+
+The update process detected that you were using @PRODUCT_NAME@Service
+to start your connections, and moved all your config files from config
+to the config-auto directory.
```

Comment:
Telling the user about change of location is good, but should we "move" the files or "copy". I thought "copy" was less destructive and good enough?

Reply:
Perhaps I'm too eager to clean things. :)

Being a copy or move, to be triggered, it has to be bound to an MSI component (file or registry value). Therefore, I introduced the `Where are my config files.txt` to serve as an anchor, and it all turned out quite naturally.

@mattock, we need the third judge to decide "copy or move" config files from `C:\Program Files\OpenVPN\config` to `C:\Program Files\OpenVPN\config-auto` when updating an OpenVPN installation that is using `openvpnserv2.exe`?

Reply:
@rozmansi there will be confusion no matter what we do:

- Copy: "why don't my changes to config <n> in config directory stick?"
- Move: "where did my configs go?"

Personally I think the latter is the smaller evil. It won't be that difficult for a user to figure out that OpenVPN is working -> config files must be somewhere -> go looking -> find config-auto -> look inside.